aiven/resource_flink_table.go

```diff
@@ -0,0 +1,160 @@
+package aiven
+
+import (
+	"context"
+
+	"github.com/aiven/aiven-go-client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var aivenFlinkTableSchema = map[string]*schema.Schema{
+	"project": {
+		Type:        schema.TypeString,
+		Description: "Project to link the Flink Table to",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"service_name": {
+		Type:        schema.TypeString,
+		Description: "Service to link the Flink Table to",
+		Required:    true,
+		ForceNew:    true,
+	},
+	"table_id": {
+		Type:        schema.TypeString,
+		Description: "Table id of the flink table",
+		Computed:    true,
+		ForceNew:    true,
```

Comment:
Looks like we can remove `ForceNew` from the `Computed` fields, here and below.

Reply:
true, i will change that